closure/goog/useragent/useragent.js

```diff
@@ -207,8 +194,12 @@ goog.userAgent.SAFARI = goog.userAgent.WEBKIT;
  * @private
  */
 goog.userAgent.determinePlatform_ = function() {
-  var navigator = goog.userAgent.getNavigator();
-  return navigator && navigator.platform || '';
+  const nav = goog.userAgent.getNavigator();
+  if (nav === null || !('platform' in nav)) {
```

Comment:
To be honest, I'd rather see `=== null` that `goog.isNull`, though there's disagreement among the maintainers about this.  In any case, I don't see any reason to make this less robust to the possibility of `navigator` being `undefined`.  A truthiness check seems perfectly appropriate here.